src/redux/configureStore.js

```diff
@@ -0,0 +1,12 @@
+import { configureStore, combineReducers } from 'redux';
```

Comment:
- [ ] You should import `configureStore` from `@reduxjs/toolkit`, not `redux`. Kindly fix that.